### DISCUSSION TITLE
When i upload image getting console error and its not store in local storage?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4392] [Q&A]
Hi @artf ..
     When i add 3mb size of image getting console error and its not stored in localstorage also... is there way to increase the localstorage size using javascript code?
     
     
![image](https://user-images.githubusercontent.com/50009624/174060932-641094c6-3849-4707-b63c-3c2e48988dae.png)

**Comment by artf** [comment]
The demo uses the default localStorage which has a size limit (~5mb). If this is something related to your project I'd suggest using [grapesjs-indexeddb](https://github.com/artf/grapesjs-indexeddb)

**Reply by senthil177cse** [reply]
Hi @artf ...
    I tried to add indexeddb into my project but getting error in node_modules/grapesjs-indexeddb/dist/index.d.ts file... i have attached below error screenshot and my code declaration screenshot also.. please check and give me solution for that..
    
![image](https://user-images.githubusercontent.com/50009624/174206632-dc84011e-b79a-478f-8dea-96bb2879768e.png)
![image](https://user-images.githubusercontent.com/50009624/174206669-643c2b1a-1da4-4d27-a9ee-04eed18a39a9.png)
   
**### Code declaration screenshot below:**
![image (68)](https://user-images.githubusercontent.com/50009624/174206828-95b57018-a7f4-4fb2-9db1-58d6b1f20de8.jpg)
![image (18)](https://user-images.githubusercontent.com/50009624/174207189-be4434c5-9468-4f8a-921d-be6e461e3f2a.png)
